audit/projects/k8s-infra-public-pii/iam.json

```diff
@@ -1,5 +1,11 @@
 {
   "bindings": [
+    {
+      "members": [
+        "serviceAccount:service-226195303281@gcp-sa-bigquerydatatransfer.iam.gserviceaccount.com"
+      ],
+      "role": "roles/bigquerydatatransfer.serviceAgent"
```

Comment:
/cc @ameukam 
I'm assuming this is https://github.com/kubernetes/k8s.io/pull/2409 (or prep for it)

Reply:
Yep. Any attempt to use the service `bigquery-datatransfer.googleapis.com` will trigger creation of a default service agent. https://cloud.google.com/iam/docs/service-agents